bindings/iDynTree.i

```diff
@@ -19,6 +20,9 @@
 // Wrap the std::vector<std::int> params
 %template(IntVector) std::vector<int>;
 
+// Wrap the std::unordered_map<std::string, double>
+%template(StringToFloatUnorderedMap) std::unordered_map<std::string, double>;
```

Comment:
```suggestion
%template(StringToFloatUnorderedMap) std::unordered_map<std::string, double>;
```

Why not `StringToDoubleUnorderedMap` ?

Reply:
Bad Python habits :) Fixed in https://github.com/robotology/idyntree/pull/1180/commits/6ef64ebee9cc369072320603429206337012ec70.